core/gui/src/app/workspace/service/operator-menu/operator-menu.service.ts

```diff
@@ -456,14 +433,6 @@ export class OperatorMenuService {
   private getNonOverlappingPosition(position: Point, positions: Point[]): Point {
     let overlapped = false;
     const operatorPositions = positions.concat(
-      this.workflowActionService
-        .getTexeraGraph()
-        .getAllOperators()
-        .map(operator => this.workflowActionService.getOperatorGroup().getOperatorPositionByGroup(operator.operatorID)),
```

Comment:
just want to double check if this should be removed. it seems like it is getting operator position?